src/engraving/dom/engravingitem.h

```diff
@@ -259,6 +259,10 @@ class EngravingItem : public EngravingObject
 
     virtual int subtype() const { return -1; }                    // for select gui
 
+    // Index to a set of properties if the same element type can represent
+    // different set of properties
+    virtual int propset() const { return subtype(); }
```

Comment:
This is not a good solution. Since this is only relevant for Inspector, you need to solve this problem at the AbstractInspectorModel level:

1. declare a static function in AbstractInspectorModel:
`ElementKey AbstractInspectorModel::makeKey(const EngravingItem*)` -  it will create the correct key for TempoText (and other engraving items)

2. call this function from InspectorListModel::setElementList